app/services/lay_cards.rb

```diff
@@ -0,0 +1,12 @@
+class LayCards < Turn
+  ACTION = 'lay_cards'
+  VALID_FACES = ['a', '4', '5', '6', '9', 'j', 'q', 'k']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.